main.tf

```diff
@@ -21,7 +30,6 @@ module "orca" {
   rds_security_group_id    = var.rds_security_group_id
   system_bucket            = var.system_bucket
   vpc_id                   = var.vpc_id
-  workflow_config          = var.workflow_config
```

Comment:
This was the one variable tying us to Cumulus. Since we aren't using their step-function wrapper, we can remove it and associated resources.